internal/api/logout.go

```diff
@@ -15,17 +24,44 @@ func (a *API) Logout(w http.ResponseWriter, r *http.Request) error {
 
 	a.clearCookieTokens(config, w)
 
+	behavior := LogoutGlobal
+
+	switch r.URL.Query().Get("behavior") {
+	case "global":
+		behavior = LogoutGlobal
+
+	case "local":
+		behavior = LogoutLocal
+
+	case "others":
+		behavior = LogoutOthers
+
+	default:
+		return badRequestError(fmt.Sprintf("Unsupported logout behavior %q", r.URL.Query().Get("behavior")))
+	}
```

Comment:
i don't think we should define this in the query parameter - this makes it possible for a malicious actor to logout all sessions just by tweaking the query parameter

Reply:
The malicious actor would already have to have access to the user's session, at which point they are the user and can do whatever they want.

Reply:
Wouldn't this behavior (returning an error if there is no new `behavior` query parameter cause all existing logout API calls to break?

Reply:
That's a good point - if one is querying the API and not going through the client library their setup might break. Yeah we might wish to check on the # of users querying the API directly before following through

Reply:
> Wouldn't this behavior (returning an error if there is no new `behavior` query parameter cause all existing logout API calls to break?

Accident, the intent was to have global be the default (which is the current behavior). Fixed now.